airflow/contrib/operators/gcs_to_s3.py

```diff
@@ -88,11 +91,17 @@ def __init__(self,
         self.dest_s3_key = dest_s3_key
         self.dest_verify = dest_verify
         self.replace = replace
+        self.do_xcom_push = do_xcom_push
 
     def execute(self, context):
+        # this operator calls super and uses the returned value, so we set do_xcom_push to
```

Comment:
This is nasty, we should be able to return this, or pass the `do_xcom_push` to the super.